src/client-functions/selectors/selector-attribute-filter.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-undef */
```

Comment:
add our usual header comment like `this file is used both on server and client...`